New File Templates/README.md

```diff
@@ -8,10 +8,25 @@ files and their content will be dynamically generated based on the user input.
 ## How to Install
 To install a new file template, copy the `xctemplate` file to your file templates directory:
 ```
-/Users/<your username>/Library/Developer/Xcode/Templates/File\ Templates/
+~/Library/Developer/Xcode/Templates/File\ Templates/
 ```
 If you want to group related templates together under a custom name, add another subdirectory:
  ```
-/Users/<your username>/Library/Developer/Xcode/Templates/File\ Templates/Livefront
+~/Library/Developer/Xcode/Templates/File\ Templates/Livefront
 ```
 This will create a "Livefront" section at the bottom of the `New > File…` dialog.
+
+## Coordinator Template
+
+Livefront projects use the Coordinator pattern to manage navigation within applications. A coordinator 
+is responsible for receiving a `Route` enum value and executing the corresponding navigation to the
+correct screen of the app. This template creates a new coordinator file which will handle 
+navigation for a particular area or collection of screens within the app.
+
+## SwiftUI Feature Template
```

Comment:
What do you think about calling this `Pentimento SwiftUI Feature Template` or `SwiftUI Pentimento Feature Template`?